--- vendor/Twig-2.x/Twig/Node/WithNode.php	2024-03-04 14:06:02.592190603 +0300
+++ vendor/Twig-2.x/Twig/Node/WithNode.php	2025-10-30 15:36:51.740949779 +0300
@@ -20,7 +20,7 @@
  */
 class WithNode extends Node
 {
-    public function __construct(Node $body, Node $variables = null, bool $only = false, int $lineno, string $tag = null)
+    public function __construct(Node $body, Node|null $variables = null, bool $only = false, int $lineno, string|null $tag = null)
     {
         $nodes = ['body' => $body];
         if (null !== $variables) {
