--- vendor/Twig-2.x/Twig/Node/ImportNode.php	2024-03-04 14:06:02.592190603 +0300
+++ vendor/Twig-2.x/Twig/Node/ImportNode.php	2025-10-30 15:20:43.475938812 +0300
@@ -22,7 +22,7 @@
  */
 class ImportNode extends Node
 {
-    public function __construct(AbstractExpression $expr, AbstractExpression $var, int $lineno, string $tag = null, bool $global = true)
+    public function __construct(AbstractExpression $expr, AbstractExpression $var, int $lineno, string|null $tag = null, bool $global = true)
     {
         parent::__construct(['expr' => $expr, 'var' => $var], ['global' => $global], $lineno, $tag);
     }
