--- vendor/Twig-2.x/Twig/Node/Expression/Test/DefinedTest.php	2024-07-16 13:14:05.094888434 +0300
+++ vendor/Twig-2.x/Twig/Node/Expression/Test/DefinedTest.php	2026-02-10 13:30:57.351869837 +0300
@@ -35,7 +35,7 @@
  */
 class DefinedTest extends TestExpression
 {
-    public function __construct(Node $node, string $name, Node $arguments = null, int $lineno)
+    public function __construct(Node $node, string $name, Node|null $arguments, int $lineno)
     {
         if ($node instanceof NameExpression) {
             $node->setAttribute('is_defined_test', true);
