--- vendor/Twig-2.x/Twig/Node/ModuleNode.php	2024-07-16 13:14:05.094888434 +0300
+++ vendor/Twig-2.x/Twig/Node/ModuleNode.php	2026-02-10 13:27:32.673274695 +0300
@@ -30,7 +30,7 @@
  */
 class ModuleNode extends Node
 {
-    public function __construct(Node $body, AbstractExpression $parent = null, Node $blocks, Node $macros, Node $traits, $embeddedTemplates, Source $source)
+    public function __construct(Node $body, AbstractExpression|null $parent, Node $blocks, Node $macros, Node $traits, $embeddedTemplates, Source $source)
     {
         if (__CLASS__ !== \get_class($this)) {
             @trigger_error('Overriding '.__CLASS__.' is deprecated since Twig 2.4.0 and the class will be final in 3.0.', E_USER_DEPRECATED);
