--- vendor/Twig-2.x/Twig/Test/NodeTestCase.php	2024-03-04 14:06:02.596190640 +0300
+++ vendor/Twig-2.x/Twig/Test/NodeTestCase.php	2025-10-30 16:17:46.826117388 +0300
@@ -29,7 +29,7 @@
         $this->assertNodeCompilation($source, $node, $environment, $isPattern);
     }
 
-    public function assertNodeCompilation($source, Node $node, Environment $environment = null, $isPattern = false)
+    public function assertNodeCompilation($source, Node $node, Environment|null $environment = null, $isPattern = false)
     {
         $compiler = $this->getCompiler($environment);
         $compiler->compile($node);
@@ -41,7 +41,7 @@
         }
     }
 
-    protected function getCompiler(Environment $environment = null)
+    protected function getCompiler(Environment|null $environment = null)
     {
         return new Compiler(null === $environment ? $this->getEnvironment() : $environment);
     }
