--- vendor/Twig-2.x/Twig/Node/ForLoopNode.php	2024-03-04 14:06:02.592190603 +0300
+++ vendor/Twig-2.x/Twig/Node/ForLoopNode.php	2025-10-30 15:16:50.980825696 +0300
@@ -20,7 +20,7 @@
  */
 class ForLoopNode extends Node
 {
-    public function __construct(int $lineno, string $tag = null)
+    public function __construct(int $lineno, string|null $tag = null)
     {
         parent::__construct([], ['with_loop' => false, 'ifexpr' => false, 'else' => false], $lineno, $tag);
     }
