--- vendor/Twig-2.x/Twig/Node/EmbedNode.php	2024-03-04 14:06:02.592190603 +0300
+++ vendor/Twig-2.x/Twig/Node/EmbedNode.php	2025-10-30 15:15:43.499922128 +0300
@@ -23,7 +23,7 @@
 class EmbedNode extends IncludeNode
 {
     // we don't inject the module to avoid node visitors to traverse it twice (as it will be already visited in the main module)
-    public function __construct(string $name, int $index, AbstractExpression $variables = null, bool $only = false, bool $ignoreMissing = false, int $lineno, string $tag = null)
+    public function __construct(string $name, int $index, AbstractExpression|null $variables = null, bool $only = false, bool $ignoreMissing = false, int $lineno, string|null $tag = null)
     {
         parent::__construct(new ConstantExpression('not_used', $lineno), $variables, $only, $ignoreMissing, $lineno, $tag);
 
