--- vendor/Twig-2.x/Twig/Sandbox/SecurityNotAllowedMethodError.php	2024-03-04 14:06:02.596190640 +0300
+++ vendor/Twig-2.x/Twig/Sandbox/SecurityNotAllowedMethodError.php	2025-10-30 16:13:34.050702864 +0300
@@ -23,7 +23,7 @@
     private $className;
     private $methodName;
 
-    public function __construct(string $message, string $className, string $methodName, int $lineno = -1, string $filename = null, \Exception $previous = null)
+    public function __construct(string $message, string $className, string $methodName, int $lineno = -1, string|null $filename = null, \Exception|null $previous = null)
     {
         if (-1 !== $lineno) {
             @trigger_error(sprintf('Passing $lineno as a 3th argument of the %s constructor is deprecated since Twig 2.8.1.', __CLASS__), E_USER_DEPRECATED);
