--- vendor/Twig-2.x/Twig/Node/IncludeNode.php	2022-11-30 16:27:08.035961512 +0300
+++ vendor/Twig-2.x/Twig/Node/IncludeNode.php	2022-12-01 22:09:58.051888312 +0300
@@ -22,7 +22,7 @@
  */
 class IncludeNode extends Node implements NodeOutputInterface
 {
-    public function __construct(AbstractExpression $expr, AbstractExpression $variables = null, bool $only = false, bool $ignoreMissing = false, int $lineno, string $tag = null)
+    public function __construct(AbstractExpression $expr, ?AbstractExpression $variables, bool $only, bool $ignoreMissing, int $lineno, string $tag = null)
     {
         $nodes = ['expr' => $expr];
         if (null !== $variables) {
